k8s/README.md

```diff
@@ -215,15 +182,19 @@ As you can see this template is using values coming from `Chart` and `Values` fi
 Please read [The Chart Template Developer’s Guide](https://docs.helm.sh/chart_template_guide/#the-chart-template-developer-s-guide)
 to learn about creating your own charts.
 
-To route traffic to our gateway service we’ll be using ingress. For ingress to work `Ingress Controller` has to be enabled on our cluster:
+To route traffic to our gateway service we'll be using ingress. For ingress to work `Ingress Controller` has to be enabled on our cluster. Follow instructions form [Ingress Installation docs](https://kubernetes.github.io/ingress-nginx/deploy/):
 
-`$ minikube addons enable ingress`
+```sh
+$ kubectl --context=docker-desktop apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/mandatory.yaml
```

Comment:
Does the chart installation of nginx-ingress not work on docker-desktop clusters?

Reply:
Not sure what you mean. It is installed to `docker-desktop` cluster here.

Reply:
I think maybe you're asking if it would work on docker-desktop clusters... yes it works fine: https://kubernetes.github.io/ingress-nginx/deploy/#docker-for-mac ... the only restriction is that it will only listen on port 80 or 443. I think I could further simplify this tutorial by forgoing ingress altogether and setting gateway`service` definition to be of `NodePort` type which would allow to expose that service on any port we desire... which maybe would play better for people with existing `docker-desktop` clusters.

Reply:
I mean why are you installing manifest directly rather than `chart upgrade --install nginx-ingress`?

Reply:
Just tested it with ingress controller installed via helm chart and doesn't seem to play well with docker for desktop for some reason... It forces redirection to https 🤔

Reply:
This was asked mostly out of curiosity. Feel free to merge!